7_deploy_osh/roles/suse-dev-patcher/defaults/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+# defaults file for suse-dev-patcher
+
+devenv_root: /opt
+
+suse_dev_patcher_packages:
+  - autoconf
+  - git
+  - jq
+  - curl
+
+osh_repos:
+  openstack-helm-infra:
+    src: https://git.openstack.org/openstack/openstack-helm-infra
+    version: 5da44ee3095906d97808fe73b8d0f6617cd623c7  # commit ref 4th December 2018
+  openstack-helm:
+    src: https://git.openstack.org/openstack/openstack-helm
+    version: 030868aee3e14122eac1d39a1bb2d154ef9c86c9  # frozen ref 4th December 2018
+  openstack-helm-images:
+    src: https://github.com/openstack/openstack-helm-images
+    version: master
```

Comment:
could you freeze shas there?

Reply:
I did not add specific shas here as we are merging patches in and may want to leverage latest changes there instead of keep updating sha with each patch merge. I see only our team patches in upstream so we may not worry about destabilization of master branch.